### DISCUSSION TITLE
Popular Plays

### DISCUSSION THREAD

**Discussion by bn641243** [Security-Onion-Solutions/securityonion#10793] [Unsupported Versions] [answered]
Hi everybody, I'm relatively new to security onion and have two very basic questions. What are some plays that you use from the detection playbook? What are some popular plays that SOCs like implementing? Please let me know if this has already been answered in a previous discussion. Thank you!

**Comment by reyesj2** [comment, marked as answer]
The plays you would run in your environment will differ from the plays another person / org would run in their environment. We include a handful of detections from https://github.com/SigmaHQ/sigma that can be enabled / disabled at your discretion.

Here are some great free resources for you learn a bit more about playbook
https://www.youtube.com/watch?v=IS2SOlDedPc
https://docs.securityonion.net/en/2.3/playbook.html?highlight=playbook

**Reply by bn641243** [reply]
Thank you so much!
